src/pages/skills/[context]/[category]/index.tsx

```diff
@@ -130,6 +130,7 @@ const ListSkills = () => {
   const [selectedSkill, setSelectedSkill] = useState<Skill | undefined>(
     undefined
   );
+  const [categoryClicked, setCategoryClicked] = useState("");
```

Comment:
Prefer `undefined` to empty strings, in typescript you can specify it this way `useState<string | undefined>(undefined)`